src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -86,36 +85,32 @@ export const InitialCashoutView = ({
     const isDisabled = useMemo(() => {
         return (
             !_tokenValue ||
-            (!selectedBankAccount && !newBankAccount) ||
+            !isValidBankAccountNumber ||
+            isValidatingBankAccountNumber ||
             !xchainAllowed ||
             !!isBelowMinLimit ||
             !!isExceedingMaxLimit
         )
-    }, [_tokenValue, selectedBankAccount, newBankAccount, xchainAllowed, isBelowMinLimit, isExceedingMaxLimit])
+    }, [
+        _tokenValue,
+        isValidBankAccountNumber,
+        isValidatingBankAccountNumber,
+        xchainAllowed,
+        isBelowMinLimit,
+        isExceedingMaxLimit,
+    ])
```

Comment:
nit: booleans like these variables usually don't need useMemo, you're not getting any performance improvement. Also it includes potential bugs if you're missing a variable in the dependency array the the compute is broken